src/pages/auth-callback.tsx

```diff
@@ -26,16 +30,22 @@ const AuthCallbackPage: NextPageWithLayout = () => {
       const publicKey = searchParams.get('publicKey');
       const isRecovery = searchParams.get('isRecovery') === 'true';
       const redirect = searchParams.get('redirect');
+
+      // TODO: consider passing email in the callback url
```

Comment:
hey @marcinbodnar is this PR still in progress or are these TODOs intended for future work?

Reply:
It's ready, TODOs are for future works.